packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -236,6 +246,7 @@ class ThemeData with Diagnosticable {
     // properties (as well as brightness, primaryColorBrightness, and primarySwatch)
```

Comment:
Please update all instances of this comment

Reply:
I updated this one, but the other similar comments didn't reference these parameters, so I left them alone.